identity-iota/src/tangle/client.rs

```diff
@@ -185,7 +201,7 @@ impl Client {
   }
 }
 
-#[async_trait::async_trait(?Send)]
+#[async_trait::async_trait(? Send)]
```

Comment:
nit: IDE formatting thing.

```suggestion
#[async_trait::async_trait(?Send)]
```